### ISSUE TITLE
List View Should Be Added by Theme Support

### ISSUE THREAD

**Issue by spencerfinnell** [woocommerce/woocommerce-gutenberg-products-block#70]
Just adding `list-layout` to the block output will require all themes to introduce new layout styles based just on that class.

Until a theme adds extra CSS to support this new class the frontend will not properly reflect what they are previewing in the block. 

Storefront set to list view: https://cloudup.com/cCCN2gTKJDR

**Comment by claudiulodro** [comment]
List view is currently unstyled on the frontend, so it will currently just show as grid view.

My plan was to add the new styles to `woocommerce.css` in WooCommerce core. That should work great for almost all themes but would require new styles for themes that dequeue `woocommerce.css` (like Storefront).

**Comment by jameskoster** [comment]
To be honest, I'm contemplating leaving list view out of the v1. I just ran a user test and an issue came up where they switched to list view and were confused that the columns slider vanished. 

That, in addition to the theme issue, _and_ the fact that there isn't a huge amount of difference between a _list_ view and a _one column_ layout lead my to think we should consider leaving it out.

Keen to hear thoughts on that.

cc @kellychoffman @josemarques @msdesign21

**Comment by spencerfinnell** [comment]
@jameskoster I agree. WooCommerce has never had a notion of "List View" so I think it might confuse users. If a theme wants to offer a list-style view I think it makes the most sense to leverage the existing single column option.

**Comment by josemarques** [comment]
> the fact that there isn't a huge amount of difference between a list view and a one column layout lead my to think we should consider leaving it out.

If we are duplicating it a calling it a different thing I say take the simpler route and nix it.

**Comment by claudiulodro** [comment]
Good points, everyone. I'll remove it. 👍

**Comment by claudiulodro** [comment]
See #87
